site/docs/aws.md

```diff
@@ -444,6 +444,9 @@ spark-sql --packages org.apache.iceberg:iceberg-spark3-runtime:{{ versions.icebe
 [Hive](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-hive.html), [Flink](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-flink.html),
 [Trino](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-presto.html) that can run Iceberg.
 
+Recently, Amazon EMR 6.5.0 [announced](https://aws.amazon.com/about-aws/whats-new/2022/01/amazon-emr-supports-apache-iceberg/) support of Apache Iceberg's Spark 3 Runtime.
```

Comment:
Remove relative time in the doc, people might read it after many years. Let's say things in a more generic way, like Amazon EMR added Apache Iceberg in distribution since 6.5.0.

Reply:
Agreed.

it might help to think of it this way. Assume this addition to the docs might not get updated for years.


So qualifiers like `Recently` need to go (throughput). I think Jack’s phrasing makes the most sense here. If need be, I’d be happy to help with phrasing on other sections. But it should be treated like it could be read by somebody years later, where phrases like “the current release” and “recently” wouldn’t make sense then.

Reply:
Makes sense, I have pushed the changes.